### PR TITLE
Updated examples in the CQ migration guide

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 1]
```diff
@@ -166,6 +166,42 @@ INTO "example-db"."example-rp"."example-measurement"
 {{% /code-tab-content %}}
 {{< /code-tabs-wrapper >}}
 
+##### Write pivoted data to InfluxDB
+InfluxDB 1.x pivots queried fields into columns and returns tables with a column for each field.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:170]: Is there a definition of "pivot" we could link to or mention here, or is it clear in the current context? Or is this sentence itself the definition?

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 2]
```diff
@@ -166,6 +166,42 @@ INTO "example-db"."example-rp"."example-measurement"
...
+InfluxDB 2.0 does not pivot fields into columns by default, but it is possible
+with [`pivot()`](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/pivot)
+or [`schema.fieldsAsCols()`](/influxdb/v2.0/reference/flux/stdlib/influxdb-schema/fieldsascols/).
+
+If you use `to()` to write _pivoted data_ back to InfluxDB 2.0, each field columns is stored as a tag.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:175]:
```suggestion
If you use `to()` to write _pivoted data_ back to InfluxDB 2.0, each field column is stored as a tag.
```

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 3]
```diff
@@ -284,9 +320,8 @@ The CQ `RESAMPLE` clause uses data from the last specified duration to calculate
 The `EVERY` interval in `RESAMPLE` defines how often the CQ runs.
 The `FOR` interval defines the total time range queried by the CQ.
 
-To accomplish this same functionality in a Flux task, import the `experimental` package
-and use [`experimental.subDuration()`](/influxdb/v2.0/reference/flux/stdlib/experimental/subduration/)
-to set the `start` parameter in the `range()` function.
+To accomplish this same functionality in a Flux task, set the `start` parameter
+in the `range()` function to the negative `FOR` duration.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:324]: Maybe an example of what this looks like would be helpful?

**Review by pierwill — APPROVED** [review]
👍

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:170]: Good call. Updated this.

**Pull request merged**